splice_timestamp_api/src/main/java/com/splicemachine/timestamp/impl/TimestampServer.java

```diff
@@ -33,7 +33,7 @@
     /**
      * Fixed number of bytes in the message we expect to receive from the client.
      */
-    static final int FIXED_MSG_RECEIVED_LENGTH = 2; // 2 byte client id
+    static final int FIXED_MSG_RECEIVED_LENGTH = 3; // 2 byte client id
```

Comment:
should this comment be adjusted to reflect:  '2 byte client id plus refresh boolean'?